Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.h

```diff
@@ -206,11 +206,11 @@ class GstMappedOwnedBuffer : public GstMappedBuffer, public ThreadSafeRefCounted
 
 class GstMappedFrame {
     WTF_MAKE_TZONE_ALLOCATED(GstMappedFrame);
-    WTF_MAKE_NONCOPYABLE(GstMappedFrame);
+
```

Comment:
Is this safe? This means we would be copying the GstVideoFrame direclty. If we really want to make this copyable we should probably add copy operator/constructor and use `gst_video_frame_copy`, I guess

Reply:
Yeah I was wondering about that too... I think making explicit copies is better and likely safer. I'll update the PR.

Reply:
Actually I worry about the perf impact :) maybe relying on move semantics is better.